etc/skate-style/labels.mss

```diff
@@ -268,7 +268,59 @@
 // AREA LABELS
 // =====================================================================
 
-#area_label {
+@small_icon_width: 19;
+@small_icon_height: 23;
+@small_icon_transform: translate(0, -@small_icon_height / 2);
+
+@large_icon_width: 22;
+@large_icon_height: 26.63;
+@large_icon_transform: translate(0, -@large_icon_height / 2);
+
+@landmark_y_offset: 3;
+
+#area_label::icons[zoom=17][area>5000]  {
```

Comment:
Using [attachments](https://cartocss.readthedocs.io/en/latest/styling_concepts.html?highlight=attachments#attachments-and-instances) so all icons are painted before all text